waku/v2/node/wakunode2.nim

```diff
@@ -122,6 +124,9 @@ proc removeContentFilters(filters: var Filters, contentFilters: seq[ContentFilte
 template tcpEndPoint(address, port): auto =
   MultiAddress.init(address, tcpProtocol, port)
 
+template addWssFlag() =
+  MultiAddress.init(multiCodec("ws"))
```

Comment:
Shouldn't this be "wss" (codec 0x01DE) rather than "ws" (codec 0x01DD). You should also be able to initialise the MultiAddress just with the string "wss". It has an `init` function that will parse the `multiCodec` for you.